asyncapi.yaml

```diff
@@ -0,0 +1,163 @@
+asyncapi: '2.4.0'
```

Comment:
Please move this example to the `examples` folder or get rid of it. Whatever you prefer :)

Reply:
Yea we used this just for testing whether the kafka broker is working or not, we don't need this, so i am deleting it